### ISSUE TITLE
VectorQuery query string always returns vec:

### ISSUE THREAD

**Issue by avshimpi** [DAXGRID/typesense-dotnet#184]
As per TypeSense documents the vector query can have the vector field name in the vector query such as
```
{
"collection": "Chapter",
"q": "*",
"vector_query": "my_vect:([0.0012361062,-0.025194775,-0.0023046045],k=100)"
}
```
However the VectorQuery::ToQuery() always returns the object to have the property
`
"vector_query": "vec:([0.0012361062,-0.025194775,-0.0023046045],k=100)"
`
which requires the collection to have a field with the name "vec" in order for the vector search to work or else an error `Field 'vec' does not have a vector query index` is returned.

**Comment by runeanielsen** [comment]
Hi @avshimpi,

I'll take a look at it.

// Rune

**Comment by avshimpi** [comment]
Could potentially set the VectorFieldName by default to "vec" thus keeping it backwards compatible.
________________________________

**Comment by runeanielsen** [comment]
@avshimpi released in version 7.0.0

**Comment by runeanielsen** [comment]
I'll publish a new NuGet package next week.

**Comment by runeanielsen** [comment]
@avshimpi this has now been released in version 7.0.0.
